src/Config.hs

```diff
@@ -31,4 +31,4 @@ defaultConfig = Config False 14 (1000000 * 60 * 30) 5007
 -- TODO: Feel free to bikeshed the function name.
 reifyConfig
   :: ([Config -> Config], [String], [String]) -> (Config, [String], [String])
-reifyConfig = (& _1 %~ foldl' (flip id) defaultConfig)
+reifyConfig = _1 %~ foldl' (flip id) defaultConfig
```

Comment:
Out of curiosity: Is it so that the `(& ...)` construct doesn't really do anything?

Reply:
Apparently not, not fully sure why, but HLS complained so I took the Haskell way and got really lazy ;P

Reply:
Damn legit! Laziness <3 

Here, have some types!

```Haskell
λ: :t (&)
(&) :: a -> (a -> b) -> b
```

Reply:
Then it makes total sense, since we have the function we want to apply, we just apply it instead